boards/shields/zest-sensor-imu/zest_sensor_imu.overlay

```diff
@@ -17,4 +20,11 @@
 		compatible = "bosch,bme280";
 		reg = <0x76>;
 	};
-};
\ No newline at end of file
+	
+	iis2dlpc_zest_sensor_imu : iis2dlpc@33 {
+		compatible = "st,iis2dlpc";
+		reg = <0x33>; /* R19 */
+		drdy_gpios = <&sixtron_connector WKUP GPIO_ACTIVE_HIGH>; /* R18 and Rint2 */
+		drdy-int =  <1>; /* INT1*/
```

Comment:
These lines should be removed as the interrupt pin is not connected by default.

Reply:
Resolved in [4dd5c7a](https://github.com/catie-aq/zephyr_zest-sensor-imu/commit/4dd5c7aebb82b512cc8260041e4f3889dd0f67a1).